zerver/lib/push_notifications.py

```diff
@@ -364,12 +367,44 @@ def get_alert_from_message(message):
     else:
         return "New Zulip mentions and private messages from %s" % (sender_str,)
 
+def get_text_content(rendered_content):
```

Comment:
I might call this `get_mobile_push_content(rendered_content, gcm=False)`, and then we can have some `gcm` conditionals if we later start using somewhat fancier syntax.  E.g. I could imagine leaving mentions bold or something so they stand out a bit more in GCM, where that's supported.